### PR TITLE
OBPIH-3240 Enable edits of some fields in shipped PO lines

### PR DESCRIPTION
**Pull request by DominikaHatala**

### CODE REVIEW

File: grails-app/domain/org/pih/warehouse/order/OrderItem.groovy [hunk 1]
```diff
@@ -150,6 +152,12 @@ class OrderItem implements Serializable, Comparable<OrderItem> {
         }?:0
     }
 
+    Integer getQuantityInShipmentsInStandardUom() {
```
Comment (awalkowiak) [inline grails-app/domain/org/pih/warehouse/order/OrderItem.groovy:155]: This is already added in the combined shipments feature

File: grails-app/domain/org/pih/warehouse/order/OrderItem.groovy [hunk 2]
```diff
@@ -164,6 +172,10 @@ class OrderItem implements Serializable, Comparable<OrderItem> {
         return quantityReceivedInStandardUom / quantityPerUom
     }
 
+    Integer getQuantityInShipments() {
```
Comment (awalkowiak) [inline grails-app/domain/org/pih/warehouse/order/OrderItem.groovy:175]: Same as above

**Review by awalkowiak — APPROVED** [review]
LGTM, but there is some code that is already in the combined shipments feature. When we have plans on merging the combined shipments?

File: grails-app/domain/org/pih/warehouse/order/OrderItem.groovy (hunk 1 above)
Comment (DominikaHatala) [reply grails-app/domain/org/pih/warehouse/order/OrderItem.groovy:155]: But this feature branch is not yet merged to develop.

**Review by jmiranda — APPROVED** [review]

**Pull request merged**